.prettierrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  printWidth: 80,
```

Comment:
as I believe was discussed in Slack, 80 is too small 🤔

Reply:
I was following the recommendation of the official prettier website.

I have however now upped it to 140.

Reply:
For most things I'm happy to let a format tool dictate the standard, but line length is the one I feel needs customising to the project (particularly in TS which is much more verbose than JS) 😂